README.md

```diff
@@ -952,6 +952,19 @@ $ node -e "import('./index.js').then(m => console.log(m.main))"
 [Function]
 ```
 
+If the build fails with `✘ [ERROR] Could not resolve "better-sqlite3"`, you can pass extra arguments to esbuild with:
+
+- the `bundler-args` argument, for example: `--bundler-args "--external:better-sqlite3"`.
+
+… or by using the following configuration:
+
+```yaml
+package:
+  bundle:
+    extra_args:
+      - "--external:better-sqlite3"
+```
+
```

Comment:
Slight rewording to make it more general and more consistent formatting:
```suggestion
Spago does not wrap the entirety of the bundler's API (esbuild for JS builds), so it's possible to pass arguments through to it. E.g. to exclude an NPM package from the bundle you can pass the `--external` flag to esbuild:
- either through the command line, with the `--bundler-args` flag, i.e. `--bundler-args "--external:better-sqlite3"`.
- or by adding it to the configuration file:
  ```yaml
  package:
    bundle:
      extra_args:
        - "--external:better-sqlite3"
  ```

```